### ISSUE TITLE
bootstrap users

### ISSUE THREAD

**Issue by nelsonpecora** [clay/amphora#360]
Based on [this update to scratch](https://github.com/nymag/scratch-cli/pull/125), we should be able to bootstrap users from any `bootstrap.yml` file.

**Comment by nelsonpecora** [comment]
Users should be bootstrapped in the new format:

```yaml
users:
  -
    username: String
    provider: String
    auth: String
```

This should produce users that are keyed by `base64 <username@provider>` (no site prefix)

**Comment by jonwinton** [comment]
Closed by #540
